app/src/main/java/org/mozilla/focus/exceptions/ExceptionsListFragment.kt

```diff
@@ -118,9 +119,7 @@ open class ExceptionsListFragment : Fragment(), CoroutineScope {
         removeAllExceptions.setOnClickListener {
             requireComponents.trackingProtectionUseCases.removeAllExceptions()
 
-            // val domains = ExceptionDomains.load(context!!)
-            // TelemetryWrapper.removeAllExceptionDomains(domains.size)
-            // ExceptionDomains.remove(context!!, domains)
+            TelemetryWrapper.removeAllExceptionDomains()
```

Comment:
```suggestion
            val exceptions = (exceptionList.adapter as DomainListAdapter).selection()
            TelemetryWrapper.removeAllExceptionDomains(exceptions.count)
```

A bit hacky, but similar to the `ExceptionsRemoveFragment`, we could try to use the adapter size to submit in event.

Reply:
I like that! I'll do that in a follow-up. :)